prisma/schema.prisma

```diff
@@ -81,6 +81,7 @@ model User {
   availability        Availability[]
   selectedCalendars   SelectedCalendar[]
   completedOnboarding Boolean?           @default(false)
+  locale String?
```

Comment:
Maybe enforce it with a `default()`?

Reply:
My thinking here was to not add a default because when a user first signs in, it will autodetect his language and set that as default if supported. He is free to change it later.

Reply:
I am fine either way. I think English could be a good idea as default

Reply:
It's just that if it's nullable in the db it'll be something you'd have to do everywhere, but it's ok either way